packages/mux-player/src/media-theme-mux/media-theme-mux.ts

```diff
@@ -72,6 +74,8 @@ export default class MediaThemeMux extends MediaTheme {
       title: this.getAttribute('title'),
     };
 
+    props.nohotkeys = props.src == null ? true : props.nohotkeys;
```

Comment:
might be worth adding a code comment or two around these, since src <--> disabling may not be obvious to folks.